_data/locations.yml

```diff
@@ -201,3 +201,16 @@ c-base-and-online:
   accessibility:
     wheelchair: true
     toilet: true
+in-berlin:
+  short: IN-Berlin
+  name: IN-Berlin - Individual Network Berlin e.V.
+  street: "Lehrter Str. 53"
+  city: Berlin
+  osm: https://www.openstreetmap.org/node/412766878
+  web: https://www.in-berlin.de/club/room/
+  directions: |
+    - Come in! We are in the conference room.
+  accessibility:
+    wheelchair: true
+    # TODO: to be confirmed
+    #toilet: true
```

Comment:
> ♿ Rampe für Rollstuhl vorhanden, bitte erfragen. Die Räumlichkeiten sind sonst aber leider NICHT komplett barrierefrei (insbes. WC).

Source: https://belug.de/belug/location/

- [ ] @LeoniePhiline I should probably warn about that in the next announcement.